biosys/apps/main/admin.py

```diff
@@ -14,6 +16,14 @@ class MainAppAdmin(admin.ModelAdmin):
     change_list_template = 'main/main_change_list.html'
 
 
+class UserAdmin(BaseUserAdmin):
+    list_display = ('username', 'email', 'first_name', 'last_name', 'is_staff', 'auth_token')
+
+
+admin.site.unregister(User)
```

Comment:
There's an is_registered helper function that might be worth using as a check before unregistering, otherwise it will throw a NotRegistered exception.

Reply:
will do